Dockerfile-debian.template

```diff
@@ -1,105 +1,16 @@
-FROM debian:%%DEBIAN_SUITE%%
+FROM %%DISTRIBUTION%%:%%RELEASE%%
 
-# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
-RUN set -eux; \
-	groupadd -r varnish; \
-	for user in varnish vcache; do \
-		useradd -r -g varnish $user; \
-	done
+RUN apt-get update && \
+    apt-get install -y curl && \
+    curl -s https://packagecloud.io/install/repositories/varnishcache/varnish%%VARNISH_VERSION%%/script.deb.sh |  bash && \
```

Comment:
Take for example, this part. We need (mostly) reproducible builds.

https://github.com/docker-library/official-images/blob/master/README.md#repeatability